packages/common/src/util/http.js

```diff
@@ -115,7 +121,7 @@ async function readResponseBody(response, parseAs) {
     case 'stream':
       return response.body;
     default:
-      return contentType === 'application/json'
+      return contentType && contentType.includes('application/json')
```

Comment:
@josephjclark i have to use this approach because some of the response content-type are not exactly `application/json` 
For example there are cases you get a content-type `application/json; charset=utf-8`

Reply:
That's fine - thanks for calling it out!